src/sentry/web/frontend/group_tag_export.py

```diff
@@ -2,107 +2,41 @@
 
 from django.http import Http404
 
-from sentry import tagstore
 from sentry.api.base import EnvironmentMixin
-from sentry.models import Environment, EventUser, Group, get_group_with_redirect
+from sentry.models import Environment
+
+from sentry.processing.base import ProcessingError
+from sentry.processing.issues_by_tag import IssuesByTagProcessor
 from sentry.web.frontend.base import ProjectView
 from sentry.web.frontend.mixins.csv import CsvMixin
 
 
-def attach_eventuser(project_id):
-    def wrapped(items):
-        users = EventUser.for_tags(project_id, [i.value for i in items])
-        for item in items:
-            item._eventuser = users.get(item.value)
-
-    return wrapped
-
-
 class GroupTagExportView(ProjectView, CsvMixin, EnvironmentMixin):
     required_scope = "event:read"
 
     def get_header(self, key):
-        if key == "user":
-            return self.get_user_header()
-        return self.get_generic_header()
+        return tuple(IssuesByTagProcessor.get_fields(key))
 
     def get_row(self, item, key):
-        if key == "user":
-            return self.get_user_row(item)
-        return self.get_generic_row(item)
-
-    def get_generic_header(self):
-        return ("value", "times_seen", "last_seen", "first_seen")
-
-    def get_generic_row(self, item):
-        return (
-            item.value,
-            item.times_seen,
-            item.last_seen.strftime("%Y-%m-%dT%H:%M:%S.%fZ"),
-            item.first_seen.strftime("%Y-%m-%dT%H:%M:%S.%fZ"),
-        )
-
-    def get_user_header(self):
-        return (
-            "value",
-            "id",
-            "email",
-            "username",
-            "ip_address",
-            "times_seen",
-            "last_seen",
-            "first_seen",
-        )
-
-    def get_user_row(self, item):
-        euser = item._eventuser
-        return (
-            item.value,
-            euser.ident if euser else "",
-            euser.email if euser else "",
-            euser.username if euser else "",
-            euser.ip_address if euser else "",
-            item.times_seen,
-            item.last_seen.strftime("%Y-%m-%dT%H:%M:%S.%fZ"),
-            item.first_seen.strftime("%Y-%m-%dT%H:%M:%S.%fZ"),
-        )
+        fields = IssuesByTagProcessor.get_fields(key)
+        item_dict = IssuesByTagProcessor.serialize_issue(item, key)
+        return (item_dict[field] for field in fields)
```

Comment:
This is just code cleanup. It was messy to have these declared in two places. I instead used the static methods on the processor to cut a bunch of lines